python/lsst/obs/base/gen2to3/convertRepo.py

```diff
@@ -500,15 +500,28 @@ def run(self, root: str, *,
         # 'raw' dataset type).
         rootConverter.runRawIngest()
 
-        # Write curated calibrations to all calibration repositories.
+        # Write curated calibrations to all calibration runs and
+        # also in the default collection.
         # Add new collections to the list of collections the butler was
         # initialized to pass to DefineVisitsTask, to deal with the (likely)
         # case the only 'camera' dataset in the repo will be one we're adding
         # here.
         if self.config.doWriteCuratedCalibrations:
+            butler3 = Butler3(butler=self.butler3)
+            calibCollections = set()
             for run in calibs.values():
-                butler3 = Butler3(butler=self.butler3, run=run)
-                self.instrument.writeCuratedCalibrations(butler3)
+                self.instrument.writeCuratedCalibrations(butler3, run=run)
+                calibCollections.add(run)
+            # Ensure that we have the curated calibrations even if there
+            # is no calibration conversion.  It's possible that the default
+            # calib collection will have been specified (in fact the
+            # butler convert script enforces that behavior for now) so
+            # we check for the default situation
+            # Assume we know the default rather than letting
+            # writeCuratedCalibrations default itself
```

Comment:
I'm not sure that I understand the statement "Assume we know the default rather than letting writeCuratedCalibrations default itself".

Reply:
writeCuratedCalibrations has a default run where it knows to put it in instrument/calib -- here I'm using the same code to create the default in advance of calling writeCuratedCalibrations but in theory writeCuratedCalibrations could change its mind about defaulting and I might forget to fix this code to keep it in sync.